cognite/client/data_classes/data_modeling/views.py

```diff
@@ -386,17 +387,19 @@ class SingleHopConnectionDefinition(ConnectionDefinition):
     description: str | None = None
     edge_source: ViewId | None = None
     direction: Literal["outwards", "inwards"] = "outwards"
+    connection_type: Literal["multi_edge_connection"] = "multi_edge_connection"
 
     @classmethod
     def load(cls, data: dict[str, Any]) -> SingleHopConnectionDefinition:
-        output = cls(**convert_all_keys_to_snake_case(data))
-        if (type_ := data.get("type")) is not None:
-            output.type = DirectRelationReference.load(type_)
-        if (source := data.get("source")) is not None:
-            output.source = ViewId.load(source)
-        if (edge_source := data.get("edgeSource")) is not None:
-            output.edge_source = ViewId.load(edge_source)
-        return output
+        return cls(
+            type=DirectRelationReference.load(data["type"]),
+            source=ViewId.load(data["source"]),
+            name=data.get("name"),
+            description=data.get("description"),
+            edge_source=(edge_source := data.get("edgeSource")) and ViewId.load(edge_source),
+            direction=data.get("direction"),  # type: ignore[arg-type]
+            connection_type=to_snake_case(data.get("connectionType")),  # type: ignore[arg-type]
```

Comment:
These will never be null, so we shouldn't use `.get`
```suggestion
            direction=data["direction"]
            connection_type=to_snake_case(data["connectionType"])
```